Containerfile

```
@@ -9,7 +9,11 @@ ENV package_manager ${package_manager:-yum -y}
 RUN $package_manager -y install gcc git krb5-devel python3-devel python3-pip rpm-build && $package_manager -y clean all
 RUN curl --output /usr/bin/get_sources.sh https://git.centos.org/centos-git-common/raw/master/f/get_sources.sh && chmod +x /usr/bin/get_sources.sh
 # Tools required by the %prep section of some packages
-RUN $package_manager -y install \
+# PowerTools repo installed for gtk-doc
+RUN $package_manager -y install dnf-plugins-core && \
+    $package_manager -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    $package_manager config-manager --set-enabled PowerTools && \
+    $package_manager -y install \
```

Can we merge at least this one with other packages?

```suggestion
    $package_manager config-manager --set-enabled PowerTools && \
    $package_manager -y install \
    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

---

The repository needs to be installed first, then enabled :)

---

Turns out we don't even need to install it.